host-bmc/host_pdr_handler.cpp

```diff
@@ -1288,14 +1524,74 @@ void HostPDRHandler::setFRUDataOnDBus(
                     }
                 }
             }
+            else
+#endif
```

Comment:
can this endif be before else?

Reply:
If we put endif before else compilation will fail if oem-ibm is not defined.